pkg/reconciler/labeler/v1/accessors.go

```diff
@@ -63,26 +74,35 @@ func (r *revision) list(ns, name string) ([]kmeta.Accessor, error) {
 
 // patch implements accessor
 func (r *revision) patch(ns, name string, pt types.PatchType, p []byte) error {
-	_, err := r.r.client.ServingV1().Revisions(ns).Patch(name, pt, p)
+	_, err := r.client.ServingV1().Revisions(ns).Patch(name, pt, p)
 	return err
 }
 
 // configuration is an implementation of accessor for Configurations
 type configuration struct {
-	r *Reconciler
+	client              clientset.Interface
+	configurationLister listers.ConfigurationLister
 }
 
 // configuration implements accessor
 var _ accessor = (*configuration)(nil)
 
+// MakeConfigurationAccessor is a factory function to make a new configuration accessor
+func MakeConfigurationAccessor(client clientset.Interface, lister listers.ConfigurationLister) *configuration {
```

Comment:
nit: `MakeXYZ` to me always signals significant work being done to actually create the object. I'd prefer `NewXYZ` for simple constructors like this one.